<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RenderWindow</class>
 <widget class="QMainWindow" name="RenderWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1660</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RenderWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="RenderWidget" name="openGLWidget">
    <property name="geometry">
     <rect>
      <x>211</x>
      <y>0</y>
      <width>1440</width>
      <height>810</height>
     </rect>
    </property>
   </widget>
   <widget class="ListBox" name="listBox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>211</width>
      <height>600</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1660</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="actioncreatProject"/>
    <addaction name="actionopenProject"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>物体</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuLUA">
    <property name="title">
     <string>LUA</string>
    </property>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menuLUA"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="Render_action"/>
   <addaction name="separator"/>
   <addaction name="Clear_action"/>
   <addaction name="actionWirefame"/>
  </widget>
  <widget class="QDockWidget" name="dockWidget_2">
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_2">
    <widget class="QListWidget" name="listWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>256</width>
       <height>192</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <action name="actioncreatProject">
   <property name="text">
    <string>创建项目</string>
   </property>
  </action>
  <action name="actionopenProject">
   <property name="text">
    <string>打开项目</string>
   </property>
  </action>
  <action name="Render_action">
   <property name="text">
    <string>渲染</string>
   </property>
  </action>
  <action name="Clear_action">
   <property name="text">
    <string>清空</string>
   </property>
  </action>
  <action name="actionWirefame">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>线框</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>RenderWidget</class>
   <extends>QOpenGLWidget</extends>
   <header>Core/Render/renderwidget.h</header>
  </customwidget>
  <customwidget>
   <class>ListBox</class>
   <extends>QListWidget</extends>
   <header>Core/UI/listbox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
